creating_images/guidelines.adoc

```diff
@@ -11,14 +11,14 @@
 toc::[]
 
 == Overview
-When creating Docker images to run on {product-title} there are a number of best
+When creating container images to run on {product-title} there are a number of best
 practices to consider as an image author to ensure a good experience for
 consumers of those images. Because images are intended to be immutable and used
 as-is, the following guidelines help ensure that your images are highly
 consumable and easy to use on {product-title}.
 
-== General Docker Guidelines
-The following guidelines apply when creating a Docker image in general, and are
+== General Container Guidelines
```

Comment:
I know this was probably because of the simple replace, but can the above be "General Container Image Guidelines"? I think it makes a little more sense when it was Docker, but it might have lost some context with the replace.